### ISSUE TITLE
Json fail with api rest on boolean rules with carriage return

### ISSUE THREAD

**Issue by tanguyvda** [centreon/centreon#5978]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 2.8.16

**Centreon Engine version**: 1.8.1

**Centreon Broker version**: 3.0.11

**OS**: CentOS 6.9

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Steps to reproduce the issue:**
#sur l'interface, faire une boolean rule sans retour à la ligne 
#en faire une seconde en utilisant les boutons "AND" ou "OR" pour ajouter des retours à la ligne
#ou alors rajouter les retours à la ligne manuellement

#authentification à l'api
curl -X POST 'http://xxx.xxx.xxx.xxx/centreon/api/index.php?action=authenticate' -d 'username=user&password=password'

#récupération de nos booleans
curl -X POST 'http://xxx.xxx.xxx.xxx/centreon/api/index.php?action=action&object=centreon_clapi' -d '{"action":"show","object":"BOOLEANRULE"}' -H 'Content-Type: application/json' -H 'centreon_auth_token: montoken'

#cry in a corner because the json is fucked up

**Describe the results you received:**
a failed json : 
{"result":[false,"{AND}\r",false]}

**Describe the results you expected:**
a nice json 
{"result":[{"id":"5","name":"totorule","expression":"{Windows-BMA Cpu} {IS} {OK}{AND}{Windows-2012-NRPE Cpu_2} {IS} {OK}","bool_state":"1"}]}
and please, make it free from any \r \n  <br> because if you do a clapi export, you have something like that 
BOOLEANRULE;ADD;totorule;{Windows-BMA Cpu} {IS} {OK}<br/>{AND}<br/>{Windows-2012-NRPE Cpu_2} {IS} {OK};1
and I don't want to have to handle that when using the api rest ;) 

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by lpinsivy** [comment]
Hi,

Can you create the ticket on the correct forge? It's look like a Centreon BAM API issue

**Comment by tanguyvda** [comment]
No, it is on centreon web side. You can easily have the same behavior by adding a carriage return in a LDAP object description. When performing the show ldap using the REST API you'll get the following result  
`{"result":[false,false]}`
